pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -108352,7 +108352,6 @@ self: {
        description = "automatic hyphenation for Hakyll";
        license = stdenv.lib.licenses.mit;
        hydraPlatforms = stdenv.lib.platforms.none;
```

Comment:
You'll have to remove this line as well, otherwise hydra won't build this.

```suggestion
```

Reply:
Done, thanks.